membership_extension/models/membership_category.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+
+
+class MembershipCategory(models.Model):
+    _name = "membership.membership_category"
+    _description = "Membership category"
+
+    name = fields.Char(required=True, translate=True)
```

Comment:
Maybe you can add here a one2many of the membership lines as support for other operations or as query.